packages/features/bookings/lib/handleNewBooking.ts

```diff
@@ -467,13 +467,38 @@ async function handler(
           eventType.schedulingType === SchedulingType.ROUND_ROBIN &&
           eventType.rescheduleWithSameRoundRobinHost;
 
+        const userIdsSet = new Set(users.map((user) => user.id));
+
+        let routingFormResponse;
+
+        if (routedTeamMemberIds) {
```

Comment:
I think we could check for `routingFormResponseId` itself.

Reply:
My thought was that if we have a response but no `routedTeamMemberIds` then attribute matching wasn't used. Then we also don't want to use attribute weights or virtual queues